packages/core/src/messages/responses/cache-item-get-ttl.ts

```diff
@@ -1,35 +1,20 @@
 import {SdkError} from '../../errors';
 import {
+  BaseResponseError,
+  BaseResponseMiss,
   ResponseBase,
-  ResponseError,
-  ResponseHit,
-  ResponseMiss,
 } from './response-base';
+import {CacheItemGetTtlResponse} from './enums';
+
+interface IResponse {
+  readonly type: CacheItemGetTtlResponse;
```

Comment:
ItemGetType and ItemGetTtl did not previously have a `value()` method so I omitted them here too, but should all responses that follow the Hit/Miss/Error pattern provide `value()`?

Reply:
I am actually not sure if there were only specific apis that had the `value()` method, or it was all `Hit/Miss/Error` apis. The way I had been handling this was only treating the apis that had these methods as ones that needed these methods, but I didn't look closely enough and see if there was in fact a pattern of which apis had these

cc @cprice404

Reply:
My intent was this:

Any APIs where the successful response type has some data on it that a user may be trying to access after the request, the base response type should have an `| undefined` accessor. This is the only way we can provide an API where users never need to do any pattern matching on the response type if that's how they choose to structure their code.

Reply:
so it'd be great to add any that we notice are missing.